pkg/client/client.go

```diff
@@ -20,6 +20,8 @@ import (
 	"github.com/mercedes-benz/garm-operator/pkg/metrics"
 )
 
+var Instance BaseClient
```

Comment:
I'm bothered by the term `Instance` here ... Instance in the scope of GARM could be a Runner as well.

What do you think about one of the following two options:
* `Instance` will be renamed to `GarmClient`
* As `BaseClient` is an extended version of `garmClient.GarmAPI`, `BaseClient` will become `GarmClient` and with that `Instance` will be `BaseClient` (or just `Client`)

Reply:
Yes you are right, this might be confusing. I´ll probably like the second option better: then its clear that a `GarmClient` is the defact `Base` which has all necessary methods for Auth and default API methods for all garm resources. And the Instance of it is an instanciated `Client`.